server/routes/record-attendance/handlers/editAttendance.ts

```diff
@@ -89,16 +89,7 @@ export default class EditAttendanceRoutes {
       return res.redirect(`/attendance/activities/${id}/not-attended-reason`)
     }
 
-    const attendances = [
-      {
-        id: +attendanceId,
-        prisonCode: user.activeCaseLoadId,
-        status: AttendanceStatus.WAITING,
-        attendanceReason: null as AttendanceReason,
-        issuePayment: null as boolean,
-      },
-    ]
-    await this.activitiesService.updateAttendances(attendances, user)
-    return res.redirect(`/attendance/activities/${id}/attendance-list`)
+    // If not "yes" or "no", assume "reset"
+    return res.redirect(`/attendance/activities/${id}/attendance-details/${attendanceId}/reset-attendance`)
   }
```

Comment:
Wondering about the "assume reset" bit? Could be specific submit?

Reply:
I added the comment because when looking at the code it wasn't immediately obvious to me that a reset selection was being assumed.

I thought about adding an `else if` for reset, but doing that would mean there's no default return which eslint doesn't like. So it's either this or you add a reset `else if` with a default return which can never be reached.

I don't think there's any harm in assuming reset here and so long as it's clear I think it's probably the better option, but I don't have a strong opinion either way so happy to update if you disagree.